### ISSUE TITLE
Properly document available variables

### ISSUE THREAD

**Issue by smls** [markdbenson/dapper#4]
The README section on "Variables" has this to say about the `site` hash: <i>"All site variables are defined in _config.yml, and are available in layouts as [% site.\* %]"</i>.

However, there is at least one important site variable which is _not_ defined in _config.yml, namely `site.pages`. I only found out about it because a completely unrelated example in that README happens to use it to demonstrate a `for` statement. It would be nice if all the available variables were properly documented in the "Variable" section.

**Comment by olaf-beh** [comment]
Yep, would be nice. Meanwhile you can find it in Mark's source. Here are some keys/variables for the `page` hash from  [Dapper.pm](/markdbenson/dapper/blob/master/lib/App/Dapper.pm)
- `filename`
- `slug`
- `excerpt`
- `year`, `month`, ...
- ...
